cfme/js.py

```diff
@@ -7,7 +7,7 @@
 
 return {
     jquery: jQuery.active,
-    prototype: Ajax.activeRequestCount,
+    prototype: (typeof Ajax == "undefined") ? 0 : Ajax.activeRequestCount,
```

Comment:
Yup. Unless you explicitly want the `==` behavior instead of `===`, `===` should always be preferred.